sqld/src/rpc/replication_log.rs

```diff
@@ -94,7 +95,7 @@ impl<S: futures::stream::Stream + Unpin> futures::stream::Stream for StreamGuard
         self: std::pin::Pin<&mut Self>,
         cx: &mut std::task::Context<'_>,
     ) -> std::task::Poll<Option<Self::Item>> {
-        self.get_mut().s.poll_next_unpin(cx)
+        Pin::new(&mut self.get_mut().s).poll_next(cx)
```

Comment:
Out of curiosity, what is this change doing?

Reply:
I removed the `futures` crate and moved to tokio_stream (which is better but happy to elaborate on slack). The only issue is that tokio_stream's ext trait doesn't have the unpin and boxed fn so I replaced those usages. This just does the same thing as the fn does but done manually.

Reply:
Cool. No objections. I was just curious. Thanks